services/sidecar/docker/boot.sh

```diff
@@ -27,33 +26,21 @@ then
   $SC_PIP list | sed 's/^/    /'
 
 
-elif [[ ${SC_BUILD_TARGET} == "production" ]]
-then
-  DEBUG_LEVEL=info
-fi
-
 # RUNNING application ----------------------------------------
 
 # default
-DEBUG_LEVEL=info
-CONCURRENCY=2
+CONCURRENCY=1
```

Comment:
Was that also having an effect on the db stability?

Reply:
it's more to do with the CC issue I think. we discussed last week that we should set it to 1 so we do not get the problem anymore.

Reply:
yes